app/views/flash_teams/edit.html.erb

```diff
@@ -14,14 +14,14 @@
 
```

Comment:
Same here: mostly whitespace with one PR-specific change. Would be good to let the whitespace be.